scripts/activate.sh

```diff
@@ -37,7 +37,9 @@ else
     done;
 fi;
 
-read -p "NPM 2FA Code: " twofactorcode
+if [ -z "$NPM_TOKEN" ]; then
```

Comment:
This is a good start! 

We do need to pass this NPM_TOKEN env variable to the `npm dist-tag` command as well. What about something like this?

```bash
IS_NPM_OTP=false

# prompt for the npm otp code when no NPN_TOKEN env variable exists
if [ -z "$NPM_TOKEN" ]; then
    IS_NPM_OTP=true
fi;

if [ "$IS_NPM_OTP" = true ]; then
    read -p "NPM 2FA Code: " twofactorcode
fi;

for env in $envs; do
    if [ "$IS_NPM_OTP" = true ]; then
        echo npm dist-tag add $module@$version "$tag-$env" --otp="$twofactorcode";
        npm dist-tag add $module@$version "$tag-$env" --otp="$twofactorcode";
    else
        echo npm dist-tag add $module@$version "$tag-$env"
        NPM_TOKEN=$NPM_TOKEN npm dist-tag add $module@$version "$tag-$env" --otp="$twofactorcode";
    fi;
done;
```